CalibTracker/SiStripChannelGain/src/SiStripGainsPCLHarvester.cc

```diff
@@ -271,8 +271,8 @@ SiStripGainsPCLHarvester::gainQualityMonitor(DQMStore::IBooker& ibooker_, const
        if(FitMPV>0.) Gains->Fill(Gain);
  
        MPVs->Fill(FitMPV);
-       if(Thickness<0.04) MPVs320->Fill(Phi,FitMPV);
-       if(Thickness>0.04) MPVs500->Fill(Phi,FitMPV);
+       if(Thickness<0.04) MPVs320->Fill(FitMPV);
```

Comment:
hi @mmusich - does the ~same code with perhaps the same bug not need to be fixed in 
CalibTracker/SiStripChannelGain/plugins/SiStripGainFromCalibTree.cc

Reply:
hello @davidlange6, right - thanks for the reminder. Done in the new commit.